eth-supply/src/helpers.rs

```diff
@@ -0,0 +1,48 @@
+use num_bigint;
+use std::ops::Add;
+use std::ops::Mul;
+use std::ops::Sub;
+
+use substreams::scalar::BigInt;
+use substreams_ethereum::pb::eth as pbeth;
+use substreams_ethereum::pb::eth::v2::BigInt as pbBigInt;
+use substreams_ethereum::pb::eth::v2::BlockHeader;
+
+// ETH burns the base fee. So burn per block = base fee = baseFeePerGas * gasUsed
+pub fn get_eip1559_burn(header: &BlockHeader) -> BigInt {
+    return pb_bigint_to_bigint(header.base_fee_per_gas.to_owned())
+        .mul(BigInt::from(header.gas_used));
+}
+
+// If passed the genesis block it will calculate all value minted from the crowdsale
+// on that block (it doesn't come as a block BalanceChange, so we inspect the calls inside)
+// the only transaction in that block.
+pub fn get_genesis_value(block: &pbeth::v2::Block) -> BigInt {
```

Comment:
Quite similar to https://github.com/messari/substreams/pull/109/files#diff-4ad539ed90583be7660cef0adf5e3fb5f8956c713edcab22c06dc06b526ea727
network/src/block_handler.rs
supply and rewards fns 
Think it might be a good idea to add general block logic like this to a shared block_hander  tool like the one I built already in a common lib and use it like that instead?

Reply:
Oh! This is useful. I'll switch my logic to get new supply, uncles, and burnt fees to use that instead.

The genesis issuance, do you think it is worth it? It is a one off thing, only happened on the very first block. I think it is too specific of a usecase to generalize that way. If someone needs to calculate genesis issuance again they can just import `get_genesis_value` instead?